### PR TITLE
Add documentation for variables option in test scenarios.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Add documentation for variables option in test scenarios.

#### Rationale:

- Follow up from #6323

### CODE REVIEW

**Comment by redhatrises** [comment]
Why is there this separate readme for testing over the existing documentation?

**Review by JAORMX — APPROVED** [review]

**Comment by ggbecker** [comment]
> Why is there this separate readme for testing over the existing documentation?

I don't know the answer. But we have plans to incorporate it to: https://complianceascode.readthedocs.io/en/latest/ so it will be unified into a single place.

File: tests/README.md [hunk 1]
```diff
@@ -345,10 +345,13 @@ The header consists of comments (starting by `#`). Possible values are:
 - `remediation` is a string specifying one of the allowed remediation types (eg. `bash`, `ansible`, `none`).
   The `none` value means that the tested rule has no implemented remediation.
 - `templates` has no effect at the moment.
+- `variables` is a comma-separated list of XCCDF values that sets a different default value for XCCDF variables.
```
Comment (matejak) [inline tests/README.md:348]:
```suggestion
- `variables` is a comma-separated list of XCCDF values that sets a different default value for XCCDF variables in a form `<variable name>=<value>`. Typically, you use only one of `profile` or `variables` in scenario metadata - default values are effective only if the variable is not defined using a selector, which is what profiles do.
```

**Comment by matejak** [comment]
> Why is there this separate readme for testing over the existing documentation?

This works nicely on Github, as it renders the README when you enter the `tests` folder. I think that it makes sense, moreover Becker promises seamless integration into the online documentation.

**Comment by redhatrises** [comment]
tests have not bearing on this PR.

**Pull request merged**